### DISCUSSION TITLE
Add 'contentEl' to QPopupProxy

### DISCUSSION THREAD

**Discussion by ddenev** [quasarframework/quasar#14828] [Ideas / Proposals] [answered]
As a follow-up to https://github.com/quasarframework/quasar/issues/14708, please add the `contentEl` computed prop to QPopupProxy as well

**Comment by yusufkandemir** [comment, marked as answer]
It has already been implemented in form of `currentComponent`: https://github.com/quasarframework/quasar/commit/4d5f5c22510938c75e78932ce597bf68ee9c9b52
It will be included in the next Quasar release.

```ts
import { QPopupProxy } from 'quasar'
import { ref, Ref } from 'vue'

const popupRef = ref() as Ref<QPopupProxy>

// ...

const component = popupRef.value.currentComponent
component.type // 'dialog' or 'menu'
component.ref // ref to QDialog or QMenu
component.ref.contentEl // Content element of QDialog or QMenu
```

**Reply by ddenev** [reply]
thanks @yusufkandemir 

to be more precise, this is not yet implemented and will be in v2.10.2 - https://github.com/quasarframework/quasar/discussions/14749
